app/helpers/api.js

```diff
@@ -1,6 +1,36 @@
 import cheerio from 'cheerio-without-node-native';
 import config from '../config/default';
 
+/**
+ * Gets post from given query
+ * @param  {String} query The query string to search for
+ * @param  {Object} options The options for the search, see the 'searchOptions' variable in helpers/api.js
+ * @return {Promise}       Returns a promise
+ */
+
+const searchPost = (options = defaultSearchOptions, query='') => {
+
+  const searchOptions = {
+    sortByDate: false || options.sortByDate,
+    pageNumber: 1 || options.pageNumber,
+    tags: 'story' || options.tag,
```

Comment:
For these variables see my comments below:

```javascript
const searchOptions = {
  sortByDate: false || options.sortByDate, // will always be options.sortByDate or undefined as false || <anything> will always be <anything>
  pageNumber: 1 || options.pageNumber, // will always be 1
  tags: 'story' || options.tag, // will always be 'story`
}
```

Better to do change the condition for all three, e.g: `sortByDate: options.sortByDate || false`